### ISSUE TITLE
listening on ::1:8080 fails - node macOS

### ISSUE THREAD

**Issue by aviramha** [metalbear-co/mirrord#2807]
```
Webpack: Starting ...
 INFO  Starting development server...
 ERROR  Error: listen EAFNOSUPPORT: address family not supported ::1:8080
Error: listen EAFNOSUPPORT: address family not supported ::1:8080
    at Server.setupListenHandle [as _listen2] (node:net:1886:21)
    at listenInCluster (node:net:1965:12)
    at node:net:2171:7
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)
```

**Comment by aviramha** [comment]
How to reproduce this issue:
```
HOST=::1 /Users/aviramhassan/Code/mirrord/target/universal-apple-darwin/debug/mirrord exec -t deployment/ip-visit-counter -- npm run serve
New mirrord version available: 3.120.0. To update, run: `"curl -fsSL https://raw.githubusercontent.com/metalbear-co/mirrord/main/scripts/install.sh | bash"`.
To disable version checks, set env variable MIRRORD_CHECK_VERSION to 'false'.
* Running binary "npm" with arguments: ["run", "serve"].
* mirrord will target: deployment/ip-visit-counter, no configuration file was loaded
* operator: the operator will be used if possible
* env: all environment variables will be fetched
* fs: file operations will default to read only from the remote
* incoming: incoming traffic will be mirrored
* outgoing: forwarding is enabled on TCP and UDP
* dns: DNS will be resolved remotely
⠲ mirrord exec
    ✓ update to 3.120.0 available
    ✓ ready to launch process
      ✓ layer extracted
      ✓ using operator
        ✓ operator version compatible
        ✓ operator license valid
        ✓ user credentials prepared
        ✓ session started
          ✓ connected to the target
    ✓ config summary                                                            
> my-project@0.1.0 serve
> vue-cli-service serve

 INFO  Starting development server...
 ERROR  Error: listen EAFNOSUPPORT: address family not supported ::1:80
Error: listen EAFNOSUPPORT: address family not supported ::1:80
    at Server.setupListenHandle [as _listen2] (node:net:1734:21)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)

```
this issue is probably because of remote environment having `HOST`. need to exclude it

**Comment by aviramha** [comment]
issue comes from portfinder npm module that tries to bind a port on each available interface (wt..)<br>we need to hook `getifaddrs` and remove ipv6 (possibly later on retrieve ips from remote and match it to be cool)
